### PR TITLE
[MS-2821] Update SDK FAQ page

### PR DESCRIPTION
**Pull request by platrofa**
We will publish Platform SDK v5 in the future and the FAQ page needs some changes.

We removed `Working with when.js and Promises in TypeScript` page as the new Platform SDK does not use when.js and promises are more common now so it is no longer necessary to explain what a promise is.

We now document the working copy files limit.

Please do not publish the changes until we have publicly released the new SDK version. I will get back to you once we have published it.

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
#3937

File: content/apidocs-mxsdk/mxsdk/index.md [hunk 1]
```diff
@@ -13,7 +13,6 @@ Browse the reference documentation below about the Mendix Metamodel:
 
 * [Understanding the Mendix Metamodel](understanding-the-metamodel) – explains the basic components of the Mendix Metamodel as well as the domain model, microflows, pages, and apps in the Mendix Metamodel
 * [JavaScript & TypeScript Resources](javascript-typescript-resources) – summarizes TypeScript and provides links to useful resources for learning more
-* [Working with when.js and Promises in TypeScript](working-with-when-js-and-promises-in-typescript) – explains what promises are, how to use them, and how resolve common problems when using them
```
Comment (ahmed-negm) [inline content/apidocs-mxsdk/mxsdk/index.md:16]: This bullet has already been removed in the other [MR](https://github.com/mendix/docs/pull/3937)

File: content/apidocs-mxsdk/mxsdk/sdk-faq.md [hunk 2]
```diff
@@ -18,11 +18,15 @@ The Model API service is hosted in the European Union.
...
-### 1.4 What Are the Limits on Project Size?
+### 1.4 What Are the Limits on App Size?
 
-The initial project mpk file that is uploaded is not allowed to exceed 250Mb (project mpr file and other files combined).
+The initial app mpk file that is uploaded is not allowed to exceed 250Mb (app mpr file and other files combined).
 
-### 1.5 Which Parts of the Mendix App Model Can I Access with the Model API?
+### 1.5 What Are the Limits on Working Copy Size?
+
+The maximun number of files allowed in a working copy is 10k.
```
Comment (ahmed-negm) [inline content/apidocs-mxsdk/mxsdk/sdk-faq.md:27]: Typo `maximum`.

I also don't know if `10k` is formal enough to be used in the documentation, or if we should use `10,000` instead

File: content/apidocs-mxsdk/mxsdk/working-with-when-js-and-promises-in-typescript.md [hunk 3]
```diff
@@ -1,148 +0,0 @@
----
```
Comment (ahmed-negm) [inline content/apidocs-mxsdk/mxsdk/working-with-when-js-and-promises-in-typescript.md:1]: Have you checked that we are no longer referencing this file?

File: content/apidocs-mxsdk/mxsdk/index.md (hunk 1 above)
Comment (platrofa) [reply content/apidocs-mxsdk/mxsdk/index.md]: I reverted the change so we will have no merge conflicts with the other [MR](https://github.com/mendix/docs/pull/3937)

File: content/apidocs-mxsdk/mxsdk/sdk-faq.md (hunk 2 above)
Comment (platrofa) [reply content/apidocs-mxsdk/mxsdk/sdk-faq.md]: Oh right. Switched to `10,000`

**Pull request closed without merging**